feg/gateway/diameter/diameter_client.go

```diff
@@ -504,3 +508,32 @@ func getVendorSpecificApplicationIDAVPs(clientCfg *DiameterClientConfig,
 	return vendorSpecificApplicationIDs
 
 }
+
+// getHostIPAddresses gets all the IPs from a list of Ip:Port
+// Returns 127.0.0.1 if no address is passed or if address is invalid
+// Those addresses will be used at Host-IP-Address AVP on CER
+// localAddresses must have IP:Port format
+func getHostIPAddresses(localAddresses []string) []datatype.Address {
+	var localAddressesIP []datatype.Address
+	for _, localAddress := range localAddresses {
+		host, _, err := net.SplitHostPort(localAddress)
+		if err != nil {
+			glog.Errorf("Skipping %s: Could not parse as a valid IP:port. Error: %s", localAddress, err)
+			continue
```

Comment:
glog.Warningf("Could not parse '%s' as a valid IP:port. Error: %s", localAddress, err)
host = strings.TrimSpace(host)

Reply:
done.